lib/components/flare_component.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:ui';
+
+import 'component.dart';
+import 'package:flame/flare_animation.dart';
+
+class FlareComponent extends PositionComponent {
+  FlareAnimation _flareAnimation;
+
+  FlareComponent(
+      String fileName, String animation, double width, double height) {
+    FlareAnimation.load(fileName).then((loadedFlareAnimation) {
+      _flareAnimation = loadedFlareAnimation;
+
+      _flareAnimation.updateAnimation(animation);
+      _flareAnimation.width = width;
+      _flareAnimation.height = height;
+    });
+  }
+
+  @override
+  bool loaded() => _flareAnimation != null;
+
+  @override
+  void render(Canvas canvas) {
+    if (_flareAnimation != null) {
```

Comment:
you don't need to verify that here, only in the `update` method...

Reply:
Done